tests/visual/grid-toolbar.html

```diff
@@ -60,9 +60,9 @@
                 <div class="k-header k-grid-toolbar">
                     <button class="k-button">Button</button>
                     <span class="k-button-group">
-                        <button class="k-button-icon k-button k-group-start"><span class="k-icon k-i-bold"></span></button>
+                        <button class="k-button-icon k-button k-group-end"><span class="k-icon k-i-bold"></span></button>
```

Comment:
Why? The first button is the start button. If for some reason the buttons do not look correctly, it's not issue of the class names, rather, the styles. You can look at the button-group styles for a fix.